static/js/tools/map/fetcher/default_stat.ts

```diff
@@ -80,8 +80,8 @@ export function useFetchDefaultStat(
             facets: resp.data.facets,
           } as EntityObservationWrapper;
         }
-        dispatch(action);
         console.log("default stat dispatched");
+        dispatch(action);
```

Comment:
curious what the difference in order of console.log and dispatch(action) causes?

Reply:
I observed delay after dispatch, so the log here could be after the reducer action. Move it above so the log sequence is more clear